index.html

```diff
@@ -122,12 +122,12 @@ <h1>Tonic</h1>
       </div>
       
       <div class="card_info">
-        <div class="card_title">
+        <div class="card_title" id="title_2">
           <h1>Multi-Post
             Stories</h1>
         </div>
 
```

Comment:
- [ ] As previously requested, all projects and their respective modal popup should be populated from the array of objects. Remove all code meant for the project's details from the HTML file.